web/ui/react-app/src/pages/alerts/CollapsibleAlertPanel.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { FC, useState, Fragment } from 'react';
+import { Link } from '@reach/router';
+import { Alert, Collapse, Table, Badge } from 'reactstrap';
+import { Rule, RuleStatus } from './AlertContents';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+interface CollapsibleAlertPanelProps {
+  rule: Rule;
+  showAnnotations: boolean;
+}
+
+const alertColors: RuleStatus<string> = {
+  firing: 'danger',
+  pending: 'warning',
+  inactive: 'success',
+};
+
+const createExpressionLink = (expr: string) => {
+  return `../graph?g0.expr=${encodeURIComponent(expr)}&g0.tab=1&g0.stacked=0&g0.range_input=1h`;
+};
+
+const CollapsibleAlertPanel: FC<CollapsibleAlertPanelProps> = ({ rule, showAnnotations }) => {
+  const [open, toggle] = useState(false);
+
+  return (
+    <>
+      <Alert onClick={() => toggle(!open)} color={alertColors[rule.state]} style={{ cursor: 'pointer' }}>
+        <FontAwesomeIcon icon={faChevronDown} fixedWidth />
+        <strong>{rule.name}</strong> ({`${rule.alerts.length} active`})
+      </Alert>
+      <Collapse isOpen={open} className="mb-2">
+        <pre style={{ background: '#f5f5f5', padding: 15 }}>
+          <code>
+            <div>
+              name: <Link to={createExpressionLink(rule.name)}>{rule.name}</Link>
```

Comment:
This needs to go to the corresponding ALERTS timeseries that is saved when an alert is active:

```suggestion
              name: <Link to={createExpressionLink(`ALERTS{alertname="${rule.name}"}`)}>{rule.name}</Link>
```